docs/concepts/fees.md

```diff
@@ -0,0 +1,98 @@
+| Create 10 accounts                                 | 0.00002261            |
+| Deploying a contract that is ~50kb                 | 0.00002965            |
+
+**Inclusion Fee**
+
+The inclusion effort of a transaction represents the work needed for:
+
+- Including the transaction in a block
+- Transporting the transaction information from node to node
+- Verifying transaction signatures
+
+Right now, the inclusion effort is always 1.0 and the inclusion effort cost is fixed to `0.000001`. 
+
+**Surge Factor**
+
+In the future, a network surge will be applied when the network is busy due to an increased influx of transactions required to be processed or a decrease in the ability to process transactions. Right now, the network surge is fixed to `1.0`.
+
+Currently, both the inclusion fee and surge factor don’t represent any significant Flow fees, however, keep in mind this can change in the future.
+
+## Storage
+
+Each Flow account has associated storage used. The account's storage used is the byte size of all the data stored in the account's storage. Accounts also have a storage capacity, which is directly tied to the amount of Flow tokens an account has. The account can, without any additional cost, use any amount of storage up to its storage capacity. 
+
+<Callout type="warning">
+If a transaction puts an account over storage capacity, that transaction fails and is reverted. Likewise, if a transaction would drop an account's balance below 0.001 Flow tokens, which is the minimum an account can have, the transaction would also fail.
+
+</Callout>
+
+**Storage Capacity**
+
+The storage capacity of an account is dictated by the amount of FLOW it has. 
+
+<Callout type="danger">
+The **minimum amount of FLOW an account can have is 0.001**. This minimum is provided by the account creator at account creation.
+
+</Callout>
+
+The minimum account reservation ensures that most accounts won't run out of storage capacity if anyone deposits anything (like an NFT) to the account.
+
+Currently, the amount required to store 100 MB in account storage is 1 Flow. 
+
+![Screenshot 2023-08-17 at 17.27.50.png](_fees_images/Screenshot_2023-08-17_at_17.27.50.png)
+
+Please note that storing data in an account on Flow doesn’t charge tokens from the account, it just makes sure you will keep the tokens as a reserve. Once the storage is freed up you can transfer the Flow tokens. 
+
+### Storage Capacity of the Payer[](../concepts/start-here/storage.md#storage-capacity-of-the-payer)
+
+The storage capacity of the Payer of a transaction is generally computed the same way as the capacity of any other account, however, the system needs to account for the transaction fees the payer will incur at the end of the transaction. The final amount of transaction fees is not yet fully known at the step when accounts are being checked for storage compliance (if their storage used is below their storage capacity). 
```

Comment:
Update `The final amount of transaction fees is not yet fully known at the step when accounts are being checked for storage compliance (if their storage used is below their storage capacity). `

to 
`The final transaction fee amount is not fully known at this step, only when accounts are checked for storage compliance.  If their storage used is more than their storage capacity, the transaction will fail. `

I'm thinking this paragraph has to do specifically with the payer of the transaction.